### ISSUE TITLE
validate acc is very low

### ISSUE THREAD

**Issue by 2225686820** [aanna0701/SPT_LSA_ViT#14]
![image](https://user-images.githubusercontent.com/54800294/181519458-f727d88e-031b-4ba2-a761-f596b0667542.png)
Hello! Your job is very nice,but when I train the model using your code,the validate accuracy is very low. I wander why?
Is there something I need to do to the Tiny ImageNet?

**Comment by aanna0701** [comment]
In my opinion, even the train accuracy is very low, so the training is not working properly.

Did you use a custom model or change hyperparameters?

Is learning not working well in cifa10 or cifar100?

**Comment by qimg412** [comment]
hello,I met the same problem,the cifar10 and cifar 100 work well,but the acc on tiny imagenet is very bad,and I have not change your learning set.Could you help me get the acc as same as your paper?

**Comment by aanna0701** [comment]
Can you show the model summary result that is output when training starts?

2022년 9월 4일 (일) 오후 10:04, qimg412 ***@***.***>님이 작성:

**Comment by qimg412** [comment]
![1](https://user-images.githubusercontent.com/77795341/188439832-0ea096a0-5f30-4c5f-9662-3f5165a28b0f.png)
![2](https://user-images.githubusercontent.com/77795341/188439847-aff5dad5-75f2-4e0c-8b6f-2577ccb40b03.png)
![3](https://user-images.githubusercontent.com/77795341/188439856-776dafcb-6531-478f-820d-7c7911d7e466.png)
![4](https://user-images.githubusercontent.com/77795341/188439858-cd15543e-b009-4fa5-8275-ff7a9c41d4fc.png)

**Comment by qimg412** [comment]
the setting and the acc of 1 st epoch as the image show

**Comment by aanna0701** [comment]
Please try training with a learning rate of 0.003 for ViT / 0.001 for other
models.


[image: 스크린샷 2022-09-05 23.51.43 복사본.png]


2022년 9월 5일 (월) 오후 8:36, qimg412 ***@***.***>님이 작성:

**Comment by qimg412** [comment]
why the V-top 1 is very slow,I use the vit model could you tell me the symbol of the V-top and the T-top?
![1](https://user-images.githubusercontent.com/77795341/188631907-4000dfcb-e5de-4b4d-8f1e-ace1cb27ab83.png)

**Comment by Clara2333333** [comment]
hello,I met the same problem，both the train accuracy and the validate accuracy is very low. I didn't change any hyperparameters.I use "python main.py --model swin --is_LSA --is_SPT" on cifar-10 .Could you help me get the acc as same as your paper?
![Uploading 微信图片_20231120091834.png…]()

**Comment by Clara2333333** [comment]
> why the V-top 1 is very slow,I use the vit model could you tell me the symbol of the V-top and the T-top? ![1](https://user-images.githubusercontent.com/77795341/188631907-4000dfcb-e5de-4b4d-8f1e-ace1cb27ab83.png)

Hi, may I ask have you solved the problem?I met the same problem. I didn't change any hyperparameters, but both the train accuracy and the validate accuracy is very low.

**Comment by tushaarkataria** [comment]
Was anyone able to solve this issue ? I am still getting the above error ?
